hbase-client/src/main/java/org/apache/hadoop/hbase/client/AsyncRegistryFactory.java

```diff
@@ -27,8 +28,6 @@
 @InterfaceAudience.Private
 final class AsyncRegistryFactory {
 
-  static final String REGISTRY_IMPL_CONF_KEY = "hbase.client.registry.impl";
```

Comment:
First reaction is ... what sort of registry is this? Need qualifier? Or Registry is a grand notion that we'll just 'know' what it does going forward (Funny.. the registry factory doesn't say either...). nit.